CourseGrab/ViewControllers/Home/HomeTableViewCell.swift

```diff
@@ -183,6 +183,10 @@ class HomeTableViewCell: UITableViewCell {
             }
         }
     }
+    
+    func setBackgroundColor(isHighlighted: Bool){
+        isHighlighted ? (containerView.backgroundColor =  [#colorLiteral(red: 0.8064131141, green: 0.8065492511, blue: 0.8063951135, alpha: 1)].first) : (containerView.backgroundColor = .white)
+    }
```

Comment:
I feel like I commented this before but idk where it went. This is better:

```suggestion
    var isHighlighted: Bool {
        didSet {
            containerView.backgroundColor =  isHighlighted ? #colorLiteral(red: 0.8064131141, green: 0.8065492511, blue: 0.8063951135, alpha: 1) : .white
        }
    }
```

Also, do we use color literals anywhere else? If not, I would avoid breaking the convention.

Reply:
Sadge